src/utils/api.js

```diff
@@ -19,3 +19,11 @@ export const getArticle = (setArticle, article_id) => {
     }).catch((err) => {
     })
 }
+
+export const getComments = (setComments, article_id) => {
+    return api.get(`/articles/${article_id}/comments`)
+    .then(({data}) => {
+        setComments(data.comments)
+    }).catch((err) => {
```

Comment:
You dont want the catch here - you need it at the end of the chain, which continues in the app. 
At the moment, you've got two catches.